kalite/i18n/tests/create_dummy_language_pack_tests.py

```diff
@@ -0,0 +1,62 @@
+import accenting
+import requests
+import zipfile
+from cStringIO import StringIO
+from mock import patch, MagicMock, call, mock_open
+
+from kalite.i18n import get_language_pack_url
+from kalite.i18n.management.commands import create_dummy_language_pack as mod
+from kalite.testing.base import KALiteTestCase
+
+
+class CreateDummyLanguagePackCommandTests(KALiteTestCase):
+    pass
+
+
+class CreateDummyLanguagePackUtilityFunctionTests(KALiteTestCase):
+
+    @patch.object(requests, "get", autospec=True)
+    def test_download_language_pack(self, get_method):
+
+        # so the next n lines before a newline separator are all about
+        # creating a dummy zipfile that is readable by zipfile.ZipFile
+        dummy_file = StringIO()
+        zf = zipfile.ZipFile(dummy_file, mode="w")
+        zf.close()
+        get_method.return_value.content = dummy_file.getvalue()  # should still be convertible to a zipfile
```

Comment:
I suspect the 2.6 error is because `get` returns a response object, so it should be something like this
```python
get_method.return_value = resp_mock = MagicMock()
resp_mock.content = dummy_file.getvalue()
```